frontend/src/employee-frontend/components/finance-basics/ServiceNeedsSection.tsx

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2017-2022 City of Espoo
```

Comment:
Doesn't matter much but the new files could have `-2024`. Maybe on tech cafe we could discuss if all these could be simply changed to `2017-Present`.